framework/generated/vulkan_generators/vulkan_struct_to_string_body_generator.py

```diff
@@ -87,6 +87,7 @@ def beginFile(self, genOpts):
             #include "util/custom_vulkan_to_string.h"
             #include "generated_vulkan_struct_to_string.h"
             #include "generated_vulkan_enum_to_string.h"
+            #include "decode/custom_vulkan_ascii_consumer.h"
```

Comment:
Perhaps `HandleIdToString` should be refactored out of that header rather than taking the hit for a new include to get it here.

Reply:
Investigating this led me to raising this issue and suggesting a fix for it in a PR: https://github.com/LunarG/gfxreconstruct/issues/959